contracts/dev-utils/generated-artifacts/DevUtils.json

```diff
@@ -0,0 +1,920 @@
+{
```

Comment:
All of these artifacts and generated wrappers should be in the top level `gitignore` and should not be committed.